app/src/App.js

```diff
@@ -238,7 +238,7 @@ const App = () => {
       </a>
       <div className="flex">
         <div className="flex-1" />
-        <div className="min-w-4xl max-w-4xl flex-2 min-h-full z-10 bg-white rounded border flex flex-col pt-6 pb-10 px-10">
+        <div className="min-w-4xl max-w-4xl flex-2 min-h-full z-10 bg-white rounded-lg border border-gray-7 flex flex-col pt-6 pb-10 px-10">
```

Comment:
can we match these in the `completed.js` component and `missing-url.js` component as well?

Reply:
will do!

Reply:
Done